cli/cli/commands/enclave/inspect/files_artifacts.go

```diff
@@ -17,7 +18,7 @@ const (
 )
 
 func printFilesArtifacts(ctx context.Context, kurtosisCtx *kurtosis_context.KurtosisContext, enclaveInfo *kurtosis_engine_rpc_api_bindings.EnclaveInfo, showFullUuids bool, _ bool) error {
-	enclaveContext, err := kurtosisCtx.GetEnclaveContext(ctx, enclaveInfo.GetName())
+	enclaveContext, err := kurtosisCtx.GetEnclaveContextFromEnclaveInfo(ctx, enclaveInfo)
```

Comment:
`GetEnclaveContext` makes a call to `GetEnclave` which is unnecessary since `enclaveInfo` is already provided in the params to this function. using `GetEnclaveContextFromEnclaveInfo`